nodebuilder/tests/swamp/config.go

```diff
@@ -20,7 +20,7 @@ type Components struct {
 // In addition, the empty block interval is set to 50ms
```

Comment:
nit: can you remove this comment pls

Reply:
:+1: https://github.com/celestiaorg/celestia-node/pull/1240/commits/47ca95443d6422af49ac4678844d5cb61f04693c